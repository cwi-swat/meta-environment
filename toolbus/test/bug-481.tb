# Reported to cause segmentation fault since communication partner cache was
# introduced (Bug #481)

process P is
  snd-msg("p")

process Q is
  rec-msg("p")

toolbus(P, Q)
