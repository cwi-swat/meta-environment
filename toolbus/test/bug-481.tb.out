line 5: First argument of snd-msg or rec-msg should be a function application
line 8: First argument of snd-msg or rec-msg should be a function application
ToolBus: execution stopped due to the above error(s)
